codeforamerica/vita-min
# Nj 121 gubernatorial

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/121

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- Just before review, add gubernatorial elections fund contribution for self and, if mfj, spouse. Add to review, xml, pdf.
- Fix accessibility of review screen and update end to end test to go all the way through the minimal intake. Add some a11y checks for the review screen itself

## How to test?
- Choose mfj
- Answer yes or no to gubernatorial elections fund questions
- Go back and edit from review screen to test that that works
- See that answers are reflected in PDF and xml

## Screenshots (for visual changes)
![Screenshot 2024-11-22 at 13-34-22 The Gubernatorial Elections Fund helps qualified New Jersey governor candidates who need financial help  FileYourStateTaxes](https://github.com/user-attachments/assets/02ea82ad-339a-4121-8f30-2698cd2af272)
